source/docs/romi-robot/getting-to-know-romi.rst

```diff
@@ -23,7 +23,9 @@ The Romi has the following built-in hardware/peripherals:
 - 1x Inertial Measurement Unit (IMU)
 - 3x LEDs (green, yellow, red)
 - 3x pushbuttons (marked A, B, and C)
-- 5x configurable GPIO channels
+- 5x configurable GPIO channels (EXT)
+
+.. note:: The built-in Buzzer is currently not supported.
```

Comment:
Isn't the buzzer used by the romi firmware? Perhaps rather then saying not supported say it's not user programmable.

Reply:
From what I could find, the buzzer is not supported by WPILib. Yes, it is physically on the control board and supported with Arduino, so I see how this isn't completely clear. I will restate it.